docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -0,0 +1,335 @@
+#. Begin simulation
+
+
+See below for more detailed information on how to assign several of these items. 
+
+.. note::
+
+  In later waves of the model, we will add further items that will need to be assigned at initialization including pre-pregnancy BMI, blood pressure and/or hemoglobin levels. We will add further documentation covering how to assign these items at that time. 
+
+.. todo::
+
+   Add information on assigning ANC and intervention propensities, if correlation is included, etc. 
+
+
+*During simulation:*
+
+Run simulants through the pregnancy model as outlined in the :ref:`MNCNH Portfolio project <2024_concept_model_vivarium_mncnh_portfolio>` page. Record all data outlined on the above page. Then move to the intrapartum model. 
+
+Pregnancy term lengths
+~~~~~~~~~~~~~~~~~~~~~~~
+
+At the beginning of pregnancy, it should be determined whether the pregnancy will be partial term or full term according to the probabilities in the table below.
+
+.. list-table:: Pregnancy term lengths probabilities
+  :header-rows: 1
+
+  * - Term length
+    - Probability
+    - Note
+  * - Partial term
+    - (incidence_c995 + incidence_c374) / (ASFR + ASFR * SBR + incidence_c995 + incidence_c374)
+    - 
+  * - Full term
+    - 1 - probability_partial_term
+    - 
+
+Sex of infant
+~~~~~~~~~~~~~~~
+
+For pregnancies that result in live birth or stillbirth outcomes, infant sex should be determined and recorded acording to the probability of male sex shown in the table below (probability of female birth is equal to 1 minus the probability of male birth). This should be performed at the start of pregnancy (transition from np to p states) or upon initialization into the p state. These sex ratios were calculated using the live births by sex 2020 GBD covariate (ID 1106), `shown here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/Live%20births%20by%20sex.ipynb>`_. Note that there is no variation by draw in this parameter. 
+
+.. _sex_ratio_table_mncnh:
+
+.. list-table:: Probability of male birth
+    :header-rows: 1
+
+    *   - Location
+        - Location ID
+        - Value
+    *   - Pakistan 
+        - 165
+        - 0.514583
+    *   - Nigeria
+        - 214
+        - 0.511785 
+    *   - Ethiopia
+        - 179
+        - 0.514271  
+
+Birthweight and Gestational Age
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A duration of pregnancy value will need to be assigned to all pregnancies regardless of the pregnancy outcome. This value will inform the duration that the simulant remains in the pregnancy state prior to transitioning to the postpartum state.
+
+For partial term pregnancies (that result in abortion/miscarriage/ectopic pregnancy), assign a duration of pregnancy sampled from a uniform distribution beween 6 and 24 weeks (individual heterogeneity with no parameter uncertainty). For these simulants, the birthweight can be assigned as N/A since they will not be going through the intrapartum model.
```

Comment:
Will we be using the duration of partial term pregnancies for anything? Maybe YLDs of maternal disorders? I'm not sure what the implications of the uniform distribution are since I don't know how we're using this value.

Reply:
Added to do note about this to come back after we assess YLDs as discussed